### PR TITLE
docs: notice of separated ingress being deprecated

### PR DESCRIPTION
**Pull request by jessesimpson36**
### Which problem does the PR fix?

this is for https://github.com/camunda/camunda-platform-helm/issues/1838

As part of the notices for deprecation inside the following documentation PR, https://github.com/camunda/camunda-docs/pull/4195 we also want to put in a comment in values.yaml so that it's visible that separated ingress will be deprecated.

this deprecation is in 8.6, and we will remove the functionality in 8.7.

### What's in this PR?

a values.yaml comment for every ingress option except global.ingress and zeebeGateway.ingress

### Checklist

Please make sure to follow our [Contributing Guide](../blob/main/docs/contributing.md).

**Before opening the PR:**

- [x] In the repo's root dir, run `make go.update-golden-only`.
- [x] There is no other open [pull request](../pulls) for the same update/change.
- [ ] Tests for charts are added (if needed).
- [ ] In-repo [documentation](../blob/main/docs/contributing.md#documentation) are updated (if needed).

**After opening the PR:**

- [x] Did you sign our CLA (Contributor License Agreement)? It will show once you open the PR.
- [ ] Did all checks/tests pass in the PR?

### CODE REVIEW

**Review by drodriguez-305 — APPROVED** [review]
Confirmed wording and all components. LGTM

File: charts/camunda-platform-alpha/values.yaml [hunk 1]
```diff
@@ -529,7 +529,7 @@ console:
     automountServiceAccountToken: false
 
   ingress:
-    ## @param console.ingress.enabled if true, an ingress resource is deployed with the Console deployment. Only useful if an ingress controller is available, like nginx.
+    ## @param console.ingress.enabled if true, an ingress resource is deployed with the Console deployment. Only useful if an ingress controller is available, like nginx. Warning, separated ingress is deprecated and will be removed in the next release, please use global.ingress instead.
```
Comment (theburi) [inline charts/camunda-platform-alpha/values.yaml:532]: Warning: Separated ingress is deprecated and will be removed in the next release. Please, use Combined ingress by setting global.ingress: true. For more information, see https://docs.camunda.io/docs/self-managed/setup/guides/ingress-setup/

File: charts/camunda-platform-alpha/values.yaml [hunk 2]
```diff
@@ -1324,7 +1324,7 @@ operate:
     automountServiceAccountToken: false
 
   ingress:
-    ## @param operate.ingress.enabled if true, an ingress resource is deployed with the Operate deployment. Only useful if an ingress controller is available, like nginx.
+    ## @param operate.ingress.enabled if true, an ingress resource is deployed with the Operate deployment. Only useful if an ingress controller is available, like nginx. Warning, separated ingress is deprecated and will be removed in the next release, please use global.ingress instead.
```
Comment (theburi) [inline charts/camunda-platform-alpha/values.yaml:1327]: Warning: This parameter to configure Separated ingress is deprecated and will be removed in the next release. Please use Combined ingress by setting global.ingress: true.

Comment (theburi) [reply charts/camunda-platform-alpha/values.yaml:1327]: let apply this text to all other component parameters

**Pull request merged**